derive/Cargo.toml

```diff
@@ -10,11 +10,7 @@ edition = "2018"
 proc-macro = true
 
 [dependencies]
-syn = { version = "0.15.31", features = [ "full", "visit" ] }
-quote = "0.6"
-proc-macro2 = "0.4"
-proc-macro-crate = "0.1.3"
-
-[features]
-default = ["std"]
-std = []
+syn = { version = "1.0", features = [ "full", "visit" ] }
+quote = "1.0"
+proc-macro2 = "1.0"
+proc-macro-crate = "0.1"
```

Comment:
Incorrect. There was a reason why we required `0.1.3`.

Reply:
and this